cmd/atlas/main.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
+	Execute()
+}
```

Comment:
Let's merge the root and the main files into one. I don't see any reason for keeping a file with 4 lines of code.

Reply:
👍